test/controllers/api/v2/users_controller_test.rb

```diff
@@ -94,6 +94,7 @@ def setup
 
     mod_user = User.unscoped.find_by_id(user.id)
     assert mod_user.mail == "john@example.com"
+    assert_redirected_to users_path
```

Comment:
This is test for API controller, so expecting redirect is probably not what we want here :)

Reply:
:see_no_evil: